fwb-plugin/src/main/java/sub/fwb/api/ImporterStepRunTests.java

```diff
@@ -32,11 +44,56 @@ public void execute(Map<String, String> params) throws Exception {
 			out.println();
 			out.println("WARNING in " + fail.getTestHeader() + ": " + fail.getMessage());
 		}
+		clearCache();
 	}
 
 	@Override
 	public String getStepDescription() {
 		return "Testabfragen in Solr";
 	}
+	
+	/**
+	 * Clearing the FWB API cache after finish indexing
+	 * @throws IOException
+	 */
+	private void clearCache() throws IOException {
+		try {
+			String webPage = "https://fwbdev.sub.uni-goettingen.de/api/cache/clear";
+			String name = "FWB";
+			String password = env.solrPassword();
+
+			String authString = name + ":" + password;
+			byte[] authEncBytes = Base64.encodeBase64(authString.getBytes());
+			String authStringEnc = new String(authEncBytes);
+			
+			URL url = new URL(webPage);
+			HttpURLConnection urlConnection = (HttpURLConnection) url.openConnection();
+			urlConnection.setRequestProperty("Authorization", "Basic " + authStringEnc);
+			InputStream is = urlConnection.getInputStream();
+			InputStreamReader isr = new InputStreamReader(is);
+			int responseCode = urlConnection.getResponseCode();
+			if (responseCode == HttpURLConnection.HTTP_OK) { // success
```

Comment:
Please use the strict-equals-operator `===` for type safety

Reply:
Java did not support this operator. Because by default it strictly check type as compare to other languages like Javascript, PHP.